src/numpy-stubs/ma/core.pyi

```diff
@@ -191,8 +191,8 @@ __all__ = [
 _UFuncT_co = TypeVar("_UFuncT_co", bound=np.ufunc, default=np.ufunc, covariant=True)
 _ShapeT = TypeVar("_ShapeT", bound=tuple[int, ...])
 _ShapeT_co = TypeVar("_ShapeT_co", bound=tuple[int, ...], default=tuple[int, ...], covariant=True)
-_DTypeT = TypeVar("_DTypeT", bound=np.dtype)
-_DTypeT_co = TypeVar("_DTypeT_co", bound=np.dtype, default=np.dtype, covariant=True)
+_DTypeT = TypeVar("_DTypeT", bound=np.dtype[Any])
+_DTypeT_co = TypeVar("_DTypeT_co", bound=np.dtype[Any], default=np.dtype[Any], covariant=True)
```

Comment:
No need for `[Any]`, in NumType the `dtype` type parameter defaults to `Any` (PEP 696)

Reply:
FYI, I also plan to change this in Numpy itself, for the 2.3 release. So that's why I didn't document it as a difference.